Misc/NEWS.d/next/Core and Builtins/2019-05-27-18-00-19.bpo-26423.RgUOE8.rst

```diff
@@ -0,0 +1 @@
+Fix possible overflow in ``wrap_lenfunc()``.
```

Comment:
Would you mind that the issue only affects systems where sizeof(long) < sizeof(size_t): only 64-bit Windows if I understand correctly.